packages/dom/test/inject-polyfill.test.js

```diff
@@ -0,0 +1,26 @@
+import injectDeclarativeShadowDOMPolyfill from '../src/inject-polyfill';
+import { withExample } from './helpers';
+
+describe('injectDeclarativeShadowDOMPolyfill', () => {
+  let dom;
+
+  beforeEach(() => {
+    dom = withExample(`
+        <div data-percy-shadow-host>
+        <template shadowroot="open">
+                <div data-percy-shadow-host>
+                    <template shadowroot="open">
```

Comment:
`template` tags won't be loaded as shadow dom, since declarative shadow DOM is interpreted during  DOM parsing.

Reply:
we're testing if polyfill converts template tags to shadow DOM, which is happening currently.